TestON/RobotTests/ScalePOC.robot

```diff
@@ -0,0 +1,341 @@
+*** Settings ***
+Documentation     ONOS Switch Scale Test
+Suite Setup       ONOS Suite Setup    ${CONTROLLER_IP}    ${CONTROLLER_USER}
+Suite Teardown    ONOS Suite Teardown
+Library           SSHLibrary
+Library           Collections
+Library           OperatingSystem
+Library           String
+Library           RequestsLibrary
+Library           HttpLibrary.HTTP
+
+*** Variables ***
+##Grab the environment variables sources from your "cell"
+${CONTROLLER_IP}    %{OC1}
+${MININET_IP}    %{OCN}
+${CONTROLLER_USER}    %{ONOS_USER}
+${MININET_USER}    %{ONOS_USER}
+##USER_HOME used for public key
+${USER_HOME}    /home/fedora
+##ONOS_HOME is where the onos dist will be deployed on the controller vm
+${ONOS_HOME}    /opt/onos
+${RESTCONFPORT}    8181
+${LINUX_PROMPT}    $
+##SWITCHES_RESULT_FILE and JENKINS_WORKSPACE can be configurable...read overriding variables in README
+##SWITCHES_RESULT_FILE is used to plot data. you can use a jenkins post job for this or do it manually
+${SWITCHES_RESULT_FILE}    ${USER_HOME}/workspace/tools/switches.csv
+${JENKINS_WORKSPACE}    ${USER_HOME}/workspace/ONOS-Stable/
```

Comment:
The test initially failed until i created this folder. May want to note that somewhere